`#warning` breaks MSVC builds including cl2.hpp

When using MSVC one gets the following error when trying to include cl2.hpp:
```
fatal error C1021: invalid preprocessor command 'warning'
```
Note that there is a solution on [SO](https://stackoverflow.com/a/279669).

--- a/include/CL/cl2.hpp
+++ b/include/CL/cl2.hpp
@@ -15,4 +15,4 @@
 //
 
 #include <CL/opencl.hpp>
-#warning cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
+#pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")

